doc/howto/cisco.md

```diff
@@ -71,30 +66,25 @@ You must reboot after entering this command to take effect. If not, you
 will observe after 10 minutes
 of entering this command, the following message.
 
-::
 
-  %RADIUS-3-IDENTFAIL: Save of unique accounting ident aborted.
+    %RADIUS-3-IDENTFAIL: Save of unique accounting ident aborted.
 
 IOS 11.x
 --------
 
 For Cisco 11.1, you normally use
 
-::
-
-  aaa new-model
-  aaa authentication ppp radppp if-needed radius
-  aaa authorization network radius none
-  aaa accounting network wait-start radius
+    aaa new-model
+    aaa authentication ppp radppp if-needed radius
+    aaa authorization network radius none
+    aaa accounting network wait-start radius
 
 to get the Cisco to talk to a radius server.
 
 With IOS 11.3
-^^^^^^^^^^^^^
+-------------
```

Comment:
If we're going to use Markdown, we should just use `##` as the section titles everywhere.  Using random different formatting is confusing.